Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/DeveloperTab/DeveloperView.xaml

```diff
@@ -131,6 +131,12 @@
                             </GridViewColumn.CellTemplate>
                         </GridViewColumn>
 
+
```

Comment:
Would it be possible to have dynamic headers?

Reply:
Theoretically yes, thats a nice idea!
I will mark this down for the next improvement